spec/support/features/datepicker_helper.rb

```diff
@@ -47,5 +47,17 @@ def datepicker_month_and_year
       year = find(".flatpickr-calendar.open .flatpickr-current-month .numInputWrapper .cur-year").value
       month.to_s + " " + year.to_s
     end
+
+    def pick_date(calendar_selector, date_selector)
+      find(calendar_selector).click
+      select_datetime_from_datepicker date_selector
+      find("body").send_keys(:escape)
+    end
+
+    def pick_datetime(calendar_selector, datetime_selector)
+      find(calendar_selector).click
+      select_datetime_from_datepicker datetime_selector
+      find("body").send_keys(:escape)
+    end
```

Comment:
These two are exactly the same except the naming, right? It would be nicer to have only one method then. I don't know how to name it though. :wink: So my best idea is to keep the current naming but remove the double-up in code.
```suggestion
    def pick_date(calendar_selector, date_selector)
      pick_datetime(calendar_selector, date_selector)
    end

    def pick_datetime(calendar_selector, datetime_selector)
      find(calendar_selector).click
      select_datetime_from_datepicker datetime_selector
      find("body").send_keys(:escape)
    end
```

What do you think?